diff -aur dddvb-0.9.23_original/ddbridge/octonet.c dddvb-0.9.23_patched/ddbridge/octonet.c
--- dddvb-0.9.23_original/ddbridge/octonet.c	2016-04-11 18:26:36.000000000 +0200
+++ dddvb-0.9.23_patched/ddbridge/octonet.c	2016-05-27 14:27:36.023282591 +0200
@@ -25,7 +25,7 @@
 
 #include "ddbridge.h"
 #include "ddbridge-regs.h"
-#include <asm-generic/pci-dma-compat.h>
+#include <linux/pci-dma-compat.h>
 
 static int adapter_alloc = 3;
 module_param(adapter_alloc, int, 0444);
diff -aur dddvb-0.9.23_original/frontends/drxk_hard.c dddvb-0.9.23_patched/frontends/drxk_hard.c
--- dddvb-0.9.23_original/frontends/drxk_hard.c	2016-04-11 18:26:36.000000000 +0200
+++ dddvb-0.9.23_patched/frontends/drxk_hard.c	2016-05-27 14:36:28.039939664 +0200
@@ -4990,12 +4990,12 @@
 	return 0;
 }
 
-static int drxk_t_get_frontend(struct dvb_frontend *fe, struct dvb_frontend_parameters *p)
-{
+//static int drxk_t_get_frontend(struct dvb_frontend *fe, struct dvb_frontend_parameters *p)
+//{
 	//struct drxk_state *state = fe->demodulator_priv;
 	//printk("%s\n", __FUNCTION__);
-	return 0;
-}
+//	return 0;
+//}
 
 static struct dvb_frontend_ops drxk_c_ops = {
 	.info = {
@@ -5015,7 +5015,7 @@
 	.i2c_gate_ctrl = drxk_gate_ctrl,
 
 	.set_frontend = drxk_set_parameters,
-	.get_frontend = drxk_c_get_frontend,
+//	.get_frontend = drxk_c_get_frontend,
 	.get_tune_settings = drxk_c_get_tune_settings,
 
 	.read_status = drxk_read_status,
@@ -5049,7 +5049,7 @@
 	.i2c_gate_ctrl = drxk_gate_ctrl,
 
 	.set_frontend = drxk_set_parameters,
-	.get_frontend = drxk_t_get_frontend,
+//	.get_frontend = drxk_t_get_frontend,
 
 	.read_status = drxk_read_status,
 	.read_ber = drxk_read_ber,
