--- a/v4l/Makefile
+++ b/v4l/Makefile
@@ -50,13 +50,13 @@ endif	# TOPDIR
 default:: prepare firmware
 	@echo Kernel build directory is $(OUTDIR)
 	$(MAKE) -C $(OUTDIR) SUBDIRS=$(PWD) $(MYCFLAGS) modules
-	./scripts/rmmod.pl check
+#	./scripts/rmmod.pl check
 #	$(MAKE) checkpatch
 
 mismatch:: prepare firmware
 	@echo Kernel build directory is $(OUTDIR)
 	$(MAKE) -Wfatal-errors -C $(OUTDIR) SUBDIRS=$(PWD) $(MYCFLAGS) CONFIG_DEBUG_SECTION_MISMATCH=y modules
-	./scripts/rmmod.pl check
+#	./scripts/rmmod.pl check
 
 # Anything in this target shouldn't be build in parallel.
 prepare:: config-compat.h Makefile.media links
